ui/landing-page/landing-page.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"

```

Comment:
What do you suggest, i think we should use the `js` and `css` locally instead of link.
If internet is not working the page UI wont show up properly.

Reply:
I have added all the required files to the `ui/shared/vendor` folder.